OsmAnd/src/net/osmand/core/android/MapRendererContext.java

```diff
@@ -205,7 +206,7 @@ private void updateMapPresentationEnvironment(boolean forceUpdateProviders) {
 		}
 		if (!mapStyles.containsKey(rendName)) {
 			Log.d(TAG, "Style '" + rendName + "' not in cache");
-			if (mapStylesCollection.getStyleByName(rendName) == null) {
+			if (mapStylesCollection.getStyleByName(rendName) == null || IGNORE_CORE_PRELOADED_STYLES) {
```

Comment:
Unnecessary

Reply:
Still required until the corresponding changes would be committed to the core library.